siteorigin-panels.php

```diff
@@ -732,11 +732,14 @@ function siteorigin_panels_generate_css($post_id, $panels_data = false){
 			));
 		}
 
+
+		$collapse_order = !empty( $grid['style']['collapse_order'] ) ? $grid['style']['collapse_order'] : ( !is_rtl() ? 'left-top' : 'right-top' );
```

Comment:
Shouldn't `is_rtl()` still make it switch even if the collapse order was set by the user?